### PR TITLE
Observatory snapshot integration

### PR DESCRIPTION
**Pull request by ethervoid**
- [x] Metrics for observatory snapshot: overquota, number of uses, remanining quota and so on
- [x] Quota for observatory snapshot service
- [x] Soft limit for observatory snapshot
- [x] Tests for the observatory snapshot part

### CODE REVIEW

**Comment by javisantana** [comment]
as a comment, I see there is a big "data observatory" service. It's **perfectly fine** for now but inside DO we could have different set of services being charged in a different way in the future.

**Comment by ethervoid** [comment]
Yes, I have it in mind and is going to be a pain in the ass change it in
the future, but I don't know how to name this service to differentiate from
the others that we don't know what are going to do.

Any idea? Maybe we could call this do_augment or do_snapshot... @rafatower
El 20/4/2016 19:33, "javi santana" <notifications@github.com> escribió:

as a comment, I see there is a big "data observatory" service. It's *perfectly
fine* for now but inside DO we could have different set of services being
charged in a different way in the future.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/CartoDB/cartodb/pull/7197#issuecomment-212527964>

File: vendor/assets/javascripts/cartodb.mod.odyssey.uncompressed.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-// version: 3.15.8
```
Comment (rafatower) [inline vendor/assets/javascripts/cartodb.mod.odyssey.uncompressed.js]: @ethervoid I think this is a mistake to correct

**Comment by ethervoid** [comment]
@rafatower final review and deploy

**Comment by jgoizueta** [comment]
:+1:

**Pull request merged**